### PR TITLE
Fix issue #506 : GitlabCommitStatus throws error after force kill

### PR DESCRIPTION
**Pull request by sbrassely**
Fix CreationException when resuming inside script. Fixes issue #506.

### CODE REVIEW

**Comment by omehegan** [comment]
@sbrassely thanks for this PR! We will review it ASAP.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java [hunk 1]
```diff
@@ -1,33 +1,37 @@
 package com.dabsquared.gitlabjenkins.workflow;
 
-import com.dabsquared.gitlabjenkins.cause.GitLabWebHookCause;
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/AddGitLabMergeRequestCommentStep.java [hunk 2]
```diff
@@ -1,33 +1,37 @@
 package com.dabsquared.gitlabjenkins.workflow;
 
-import com.dabsquared.gitlabjenkins.cause.GitLabWebHookCause;
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/AddGitLabMergeRequestCommentStep.java]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabBuildsStep.java [hunk 3]
```diff
@@ -1,39 +1,48 @@
 package com.dabsquared.gitlabjenkins.workflow;
 
-import com.dabsquared.gitlabjenkins.gitlab.api.model.BuildState;
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabBuildsStep.java]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java [hunk 4]
```diff
@@ -1,38 +1,46 @@
 package com.dabsquared.gitlabjenkins.workflow;
 
-import com.dabsquared.gitlabjenkins.gitlab.api.model.BuildState;
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java [hunk 5]
```diff
@@ -1,27 +1,32 @@
 package com.dabsquared.gitlabjenkins.workflow;
 
-import com.dabsquared.gitlabjenkins.gitlab.api.model.BuildState;
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java [hunk 6]
```diff
@@ -94,5 +104,10 @@ public ListBoxModel doFillStateItems() {
             }
             return options;
         }
+
+		@Override
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java]: Wrong indentation.

Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java:108]: Wrong indentation.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java (hunk 1 above)
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java:3]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/AddGitLabMergeRequestCommentStep.java (hunk 2 above)
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/AddGitLabMergeRequestCommentStep.java:3]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabBuildsStep.java (hunk 3 above)
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabBuildsStep.java:3]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java (hunk 4 above)
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java:3]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java (hunk 5 above)
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStep.java:3]: You should not reorder all the imports.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java [hunk 7]
```diff
@@ -127,5 +136,10 @@ public String getFunctionName() {
         public boolean takesImplicitBlockArgument() {
             return true;
         }
+
+		@Override
```
Comment (CSchulz) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java:140]: Wrong indentation.

**Review by CSchulz — CHANGES_REQUESTED** [review]

**Comment by christ66** [comment]
👍 . I assume the import stuff happening is caused by your IDE grouping all import packages. Minor NIT, but I think still safe to merge.

**Review by mreichel — APPROVED** [review]

**Comment by Argelbargel** [comment]
@omehegan: i think this got merged into the wrong branch. Should (also) go into master ;-)

**Comment by omehegan** [comment]
@Argelbargel oh, weird. Wonder why this was submitted to that other branch. Eugh. At least I didn't mistakenly announce that this was released.

**Comment by omehegan** [comment]
@sbrassely any chance you can resubmit this as a PR to master? If not I guess I can cherry pick these into master.

**Comment by sbrassely** [comment]
@omehegan I create PR #622

**Pull request merged**